### PR TITLE
Adding monitoring, search contexts

### PR DESCRIPTION
**Pull request by emchap**

### CODE REVIEW

File: trainings/sourcegraph-101/lesson-plan.md [hunk 1]
```diff
@@ -154,7 +179,33 @@ With our query filters, you can narrow things down to find just the content you
...
+
+🔎 The trainer should show:
+
+* The example use cases on the "getting started" code monitor page
+* Creating a new code monitor using "create copy of monitor" and applying a `repo:` filter
+* The fact that monitors can be connected to email, Slack, or webhooks
+
+❗️Typically, actually going through the entire Slack flow isn't necessary; instead, you can simply speak to it and share a recording afterwards if folks want more info. 
+
+❗️If the customer is interested in email monitors, identify a point person to configure the SMTP connection on the instance; without that, it won't be possible to use them.
```
Comment (camdencheek) [inline trainings/sourcegraph-101/lesson-plan.md:198]: > it won't be possible to use them

nit: Slack and webhook monitors are usable without SMTP

**Review by camdencheek — APPROVED** [review]
Nice! Glad to see code monitors getting some love 😄

File: trainings/sourcegraph-101/lesson-plan.md (hunk 1 above)
Comment (emchap) [reply trainings/sourcegraph-101/lesson-plan.md:198]: Yep! That's why it says "email monitors". :) Is there preferred phrasing?

Comment (camdencheek) [reply trainings/sourcegraph-101/lesson-plan.md:198]: oh lol. 🤦 

Nope. Just ignore me.

Comment (emchap) [reply trainings/sourcegraph-101/lesson-plan.md:198]: Hahah, all good—thank you for taking a look, super appreciate it! :)

**Pull request merged**